import-service/src/libs/lambda.ts

```diff
@@ -0,0 +1,44 @@
+import middy from "@middy/core"
+import middyJsonBodyParser from "@middy/http-json-body-parser"
+import cors from "./corsMiddleware";
+import errorHandler from "./errorHandlerMiddleware";
+import validationMiddleware from "./validatorMIddleware";
+import loggingMiddleware from "@libs/loggingMiddleware";
+import dotenv from 'dotenv';
+
+type S3Event = "S3Event";
+type ApiGatewayEvent = "ApiGatewayEvent";
+
+type Events = S3Event | ApiGatewayEvent;
+
+interface Options<T extends Events> {
+    enableCors?: boolean,
+    event: T,
+    validator: Function | null,
+    logFormatter?: (event) => string;
+}
+
+// please be informed that middleware applying order is important
+export const middyfy = <T extends Events>(handler, options: Options<T>) => {
+    const { logFormatter, validator, enableCors } = options;
+    // @ts-ignore
+    dotenv.config( { silent: true });
```

Comment:
Why do you use `@ts-ignore` before every `dotenv.config`?

Reply:
because silent property is not documented but presented in functionality